### PR TITLE
feat(web): resort index page

### PR DESCRIPTION
**Pull request by mooyg**

### CODE REVIEW

**Comment by heybereket** [comment]
Thanks for the pull request, there are some conflicts, after you fix them, I can continue reviewing this :)

File: packages/web/src/components/navbar/Navbar.tsx [hunk 1]
```diff
@@ -11,32 +11,22 @@ import {
...
   Logout as LogoutIcon,
 } from '@components/icons';
-import React, { useEffect, useRef, useState } from 'react';
+import React, {useRef, useState } from 'react';
 import { Button } from '@components/common/Button';
 import { NavItem } from '@components/navbar/NavItem';
 import { DropdownItem } from '@components/common/DropdownItem';
 import { useGetCurrentUser } from '@lib/common/getCurrentUser';
 import { useRouter } from 'next/router';
+import useOnClickOutside from 'src/hooks/useOnClickOutside';
```
Comment (heybereket) [inline packages/web/src/components/navbar/Navbar.tsx:20]: Avoid relative paths. We use absolute paths, change this to "@hooks/useOnClickOutside", and in tsconfig.json follow the other absolute paths format to add @hooks/.

**Review by heybereket — COMMENTED** [review]
Neat work, left a piece of feedback! Besides that, looks good to me :)

**Review by HenryLeC — COMMENTED** [review]
Looks Good ty

**Pull request merged**